.eslintrc.json

```diff
@@ -15,5 +15,11 @@
     "ecmaVersion": "latest",
     "sourceType": "module"
   },
-  "plugins": ["react", "@typescript-eslint"]
+  "plugins": ["react", "@typescript-eslint"],
+  "rules": {
+    "react/display-name": "off",
+    "react/react-in-jsx-scope": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-namespace": "off"
```

Comment:
This is disabled since the code is using namesapce to define Network enums. Is there any other solutions for the Network to get rid of namespaces, since it is not recommended based on this rule:
```
15:8  error  ES2015 module syntax is preferred over namespaces  @typescript-eslint/no-namespace
```